test/runner.py

```diff
@@ -525,6 +525,17 @@ def configure():
   common.configure_test_browser()
 
 
+def cleanup_emscripten_temp():
+  """Deletes all files and directories under Emscripten
+  that look like they might have been created by Emscripten."""
+  for entry in os.listdir(shared.TEMP_DIR):
+    if entry.startswith(('emtest_', 'emscripten_')):
+      try:
+        utils.delete_dir(os.path.join(shared.TEMP_DIR, entry))
+      except Exception:
+        pass
```

Comment:
How about using `force_delete_dir` here... then I think you don't need the try/catch

Reply:
Fair enough